packages/deployment/iam/cloudformation.template.yml

```diff
@@ -481,6 +481,34 @@ Resources:
                   - s3:ListAllMyBuckets
                 Resource: '*'
 
+  ScalingRole:
```

Comment:
(Mentioned above) Needs to make it into NGAP environments.  Do we have a process for getting that in place?